pkg/virt-api/webhooks/validating-webhook/validating-webhook.go

```diff
@@ -945,10 +945,15 @@ func ValidateVirtualMachineInstanceSpec(field *k8sfield.Path, spec *v1.VirtualMa
 				podExists = true
 			}
 
+			// convert Multus to Npwgv1 network for backwards compatibility
```

Comment:
Do you do it in-place? So if I post a manifest with ```multus``` used, I still end up with ```npgwv1``` attribute and w/o ```multus``` attribute? Why do we need to do it while we still support both? We'll need to migrate them to the new name when and if we decide to drop support for ```multus``` attribute anyway, so doing it here doesn't seem of help much; and it may raise questions of why previously the same manifest result was different, in a seemingly inconsistent way. I would prefer if we honour user choice of names, and revisit migration of attributes only when we decide to go with incompatible changes.

Reply:
@dankenigsberg asked for that:

> So, at least in the schema we need both and also in the UTs.

>> Yes, that is what I attempted to mean by "canonize". We should behave alike with `multus` or >> `npwgv1`. Right after (or during?) input validation, input should be changed to `npwgv1`, >> which the bulk of the code should use exclusively.

Reply:
I requested we *behave* the same way no matter if `multus` or `npwg` is placed by the user.
@booxter suggests that we *modify* the user input, which is not too polite.

Reply:
Sorry about having misled you @michaelhenkel . I now see that my text text was badly written, I never intended to modify the `spec`.

Reply:
@dankenigsberg I could even go with modifying input, *if* we would consistently migrate all existing VMIs to the new name. We don't, so with the way PR stands right now, we would have some VMIs use ```multus``` and some ```npwg```, depending on when they were created and whether they got updated after the PR was rolled in. This is the inconsistency I am primarily concerned about. Otherwise, for me personally, the decision could go both ways - either don't ever modify posted spec; or modify all specs, for new and existing VMIs. The former should be easier to pull.